tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -108,3 +108,22 @@ Feature: full text search
       | old              |
       | new              |
       | spaces           |
+
+
+  Scenario Outline: search files inside the current folder
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world inside root" to "file1.txt"
+    And user "Alice" has created folder "/Folder"
+    And user "Alice" has uploaded file with content "hello world inside Folder" to "/Folder/file2.txt"
+    And user "Alice" has created folder "/Folder/SubFolder"
+    And user "Alice" has uploaded file with content "hello world inside SubFolder" to "/Folder/SubFolder/file3.txt"
```

Comment:
do we have step to upload multiple files at once ? Please use that if exist otherwise it good